docs/src/transaction.md

```diff
@@ -47,6 +47,7 @@ A `Transaction` requires several inputs:
 4. **Epilogue**
    Completes the execution, resulting in an updated account state and a generated zero-knowledge proof. The validity of the resulting transaction is ensured by a combination of user-defined and protocol-defined checks:
    - The account's [authentication procedure](account/code.md#authentication) is called to authorize the transaction.
+   - The transaction fee is computed and removed from the account’s vault in the chain’s native asset. See [Fees](./fees.md).
```

Comment:
```suggestion
   - The transaction fee is computed and removed from the account's vault in the chain's native asset. See [Fees](./fees.md).
```

Nit: It'd be nice to use the same apostrophe (`'` instead of `’`, i.e. a plain ASCII apostrophe) as in the rest of the docs for consistency. Depending on your system, it may have been rewritten automatically. This applies to the entire PR.